graphqlserver/src/features/User/repositories/UserRepository.ts

```diff
@@ -75,33 +85,32 @@ export class UserRepository {
     return session;
   }
 
-  async verifyEmail({
-    args: { number, code },
-  }: MutationVerifyPhoneNumberArgs): Promise<Session> {
-    const isVerified = this.verifiedPhoneNumbers.includes(number)
-      ? true
-      : await this.smsVerificationRepository.verifyCode(number, code);
+  async verifyEmail(
```

Comment:
Also, in a separate PR, similar to #3 , tests must be implemented for these resolvers.

Reply:
@Locottus